### DISCUSSION TITLE
ps_plot('V-do','ts'); cannot select points to view timing results

### DISCUSSION THREAD

**Discussion by yll1111** [alexisInSAR/EZ-InSAR#108] [Help]
When I use MATLAB to view the results of PS and SBAS fusion, the command entered with 'ts' will report an error, and the time series changes of a certain point cannot be viewed. The specific error code is attached at the end. I asked AI to give me a hint to modify the code, but it will report an error no matter how many times I modify it. The system is Ubuntu 20.04, and the MATLAB version is 2022b. I have two computers here, and the same error occurs when processing. Is there any operation I overlooked before?
```
>> ps_plot('V-do', 'ts');
Could not check if this is Envisat
Deramping computed on the fly.
**** z = ax + by+ c
111251 ref PS selected
Color Range: -29.9249 to 27.6706 mm/yr
Error using matlab.ui.control.WebComponent/get
The property String of class TextArea is not recognized.

Error ts_plot (line 18)
radiusfactor = str2num(char(get(mEditBox,'String'))); % added by david this is also not a single value but a vector

Error computing UIControl Callback.
```

**Comment by alexisInSAR** [comment]
Hi, 

There is a conflict with the Figures created by EZ-InSAR and by ps_plot. Can you close EZ-InSAR and use the command? 

Kind regards, 

Alexis

**Reply by yll1111** [reply]
Thank you for your reply, the problem is solved as you said.
